### DISCUSSION TITLE
why vlad? Why do you keep adding this stuff to make things more complicated?

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [vladmandic/sdnext#2211] [General]
![Capture](https://github.com/vladmandic/automatic/assets/140368252/3a142c67-6238-4408-9e7f-fa6312f5992b)
![Capture hgjhgjh](https://github.com/vladmandic/automatic/assets/140368252/a1d997ae-eded-4b1a-a309-797ab0d5db0e)
![Capture 5555](https://github.com/vladmandic/automatic/assets/140368252/007b172f-ed25-45d8-8fbd-df310ef9bcd9)
why do we need so many checkboxes? It Breaks the workflow, why do we need to break the wheel when it was already working?

**Comment by vladmandic** [comment]
there are only 4 - see screenshot:
![image](https://github.com/vladmandic/automatic/assets/57876960/a60885f6-3bb1-499b-859d-dcb68da60c1f)

your screenshot indicates there is something pretty wrong overall. i'd start with deleteing `ui-config.json` and take it from there.

**Reply by weirdlighthouse** [reply]
I think you are misunderstanding my post; this is not one single screenshot. It is multiple screenshots, one from text to image, another from image to image, and another from the control net.

**Comment by brunogcar** [comment]
IMHO there should be more checkboxes as it keeps UI clean, less convoluted by default,  

you can always customize on settings exactly what you want to display and make it YOUR default, 
you can make all those options expanded as it is in A1111, or just select what fits your workflow better, 

but for the vast majority those options are not used frequently

today i made a suggestion to include postprocessing, that is an option that most dont use, needs to be enabled on settings, but to not pollute UI all the time, those checkboxes to collapse and expand those modules are just great

when vlad  implement that im going suggest the same to aspect ratio, that is one feature auto always refused to include,  saying it needed to be an extension as it added to much stuff to UI, etc,  and i personally think should be a core function that is actually missing from app

but again you can save UI settings to display exactly as you need/want, change default values, whats enabled or  not, etc

**Reply by weirdlighthouse** [reply]
how do I save the UI settings?

**Reply by brunogcar** [reply]
change settings you want, to fit your workflow
exe, steps, aspect ratio, seed, second  pass,  enable  what  you want, etc customize the way you need

go to System tab 
then UI Config tab

view  changes  to display what was modified, then set new defaults

and now every time you start SD.Next  UI will  be exactly was you have set

**Reply by weirdlighthouse** [reply]
okay I am in system but where exactly is UI Config tab?
![333ji3jj](https://github.com/vladmandic/automatic/assets/140368252/95729198-356f-4c31-aa8d-6fb67918350c)
where exactly man?

**Comment by vladmandic** [comment]
![image](https://github.com/vladmandic/automatic/assets/57876960/616dfea6-397e-4db6-9cc6-c655fbccc774)

really?

**Reply by weirdlighthouse** [reply]
Okay, now I understand, thanks, Mr. Vlad.
